### ISSUE TITLE
[Question]  How i roll back the database???

### ISSUE THREAD

**Issue by bhupesh7shakya** [go-gorm/gorm.io#775]
![image](https://github.com/user-attachments/assets/fe404ac0-5d1a-4bb9-9f1a-b16bc7bc9dde)
this was my database table
for some reason i have remove the colum and 
![image](https://github.com/user-attachments/assets/1c666a75-7e51-4e22-8e65-3f120a4bbc63)
then it is not refeclting in the database its still there..!!!

**Comment by VLS-v** [comment]
Hi. This is stated in the [documentation](https://gorm.io/docs/migration.html#Auto-Migration):
> NOTE: AutoMigrate will create tables, missing foreign keys, constraints, columns and indexes. It will change existing column’s type if its size, precision, nullable changed. **It WON’T delete unused columns to protect your data**.

I hope it helps somehow.

**Comment by bhupesh7shakya** [comment]
> Hi @bhupesh7shakya. This is stated in the [documentation](https://gorm.io/docs/migration.html#Auto-Migration):
> 
> > NOTE: AutoMigrate will create tables, missing foreign keys, constraints, columns and indexes. It will change existing column’s type if its size, precision, nullable changed. **It WON’T delete unused columns to protect your data**.
> 
> I would recommend using '[golang-migrate](https://pkg.go.dev/github.com/golang-migrate/migrate/v4)' to track database changes. I hope this helps!

yep i found that i am looking for somehting like migrations like django..! is there any of it
